.github/workflows/build.yaml

```diff
@@ -45,9 +43,9 @@ jobs:
       - name: package release assets
         run: |
           mkdir _dist
-          cp containerd-shim-slight-v1/target/${{ matrix.config.arch }}-unknown-linux-musl/release/containerd-shim-*-v1 _dist/
-          cp containerd-shim-spin-v1/target/${{ matrix.config.arch }}-unknown-linux-musl/release/containerd-shim-*-v1 _dist/
-          cp containerd-shim-wws-v1/target/${{ matrix.config.arch }}-unknown-linux-musl/release/containerd-shim-*-v1 _dist/
+          for shim in slight spin wws; do
```

Comment:
Did you use the for here to constrain the shim projects for release rather than using `*` in the `cp containerd-shim-*-v1/target/....`?

Reply:
Nope I can change it to `cp containerd-shim-*-v1/target/....`